### ISSUE TITLE
[WadMerge] Bad command call: LOAD. Internal error (Valiant.wad)

### ISSUE THREAD

**Issue by jerome-trc** [MTrop/DoomTools#113]
Full log:
```
ERROR: ./wadmerge.valiant.txt, line 1: Bad command call: LOAD. Internal error.
    Caused by: IllegalArgumentException: The provided entry name, "\FLAT**", is invalid. It must be up to 8 characters long; all characters must be A-Z (uppercase only), 0-9, and [ ] - _ ^ plus the backslash \.
```

Encountered in the build distributed as doomtools-bash-2024.02.27.034322127.tar.gz. Valiant.wad is the latest distribution from idgames. To reproduce:

`LOAD valiant /path/to/Valiant.wad`

Platform:
```
$ uname -a
Linux jerome 5.15.0-100-generic #110-Ubuntu SMP Wed Feb 7 13:27:48 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by MTrop** [comment]
Fixed in commit 9eca73f8915510cd8d86b853d6a512d6a0b2e15a.

**Comment by jerome-trc** [comment]
Much appreciated.
